src/components/CalendarModal/CalendarModal.jsx

```diff
@@ -0,0 +1,54 @@
+import 'react-calendar/dist/Calendar.css'
+import Calendar from "react-calendar";
+import React, { useState } from 'react';
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCalendarAlt } from "@fortawesome/free-solid-svg-icons";
+import './calendarModal.scss'
+import Modal from '../Modal';
+import { useEffect } from 'react';
+
+export default function CalendarModal({ calendarState, title = "Date Range", iconYPosition = "" }) {
```

Comment:
The Calendar modal is looking great! There's a couple things I'd like us to add and one thing that seems like a bug I noticed:
* Can we add Cancel and OK buttons at the bottom right of the modal below the two calendars so the user can have a definite path to accept or reset the dates instead of clicking the X in the top right? The OK button would set the dates on click while the Cancel button would reset the dates back to the original dates they had when the modal opened
* There's apparently a hidden section below the calendar that changes the year when clicked:
![datepicker](https://user-images.githubusercontent.com/8600792/100184247-eb712e00-2e95-11eb-8dc8-b7d3329d373b.gif)

Reply:
A nice to have: highlighting all the days shown between the start and end dates like this image
![image](https://user-images.githubusercontent.com/8600792/100185083-d1384f80-2e97-11eb-9bc1-9af54fd43bbd.png)